src/web/components/SharingPermission/BulkAddPermissions.scss

```diff
@@ -0,0 +1,53 @@
+.bulk-add-permissions {
+  display: flex;
+  flex-direction: column;
+  border: 2px solid var(--theme-action);
+  border-radius: 5px;
+  line-height: 1;
+  box-shadow: 0px 4px 14px var(--theme-box-shadow);
+
+  .bulk-add-permissions-header {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding: 0 20px;
+    flex-direction: row;
+
+    h2 {
+      margin-top: 20px;
```

Comment:
thought about adding this to the generic styling in `text.scss` but figured this was saver for now. Happy to change the generic one from 30 to 20 though if other people are happy